src/vs/workbench/services/authentication/browser/authenticationService.ts

```diff
@@ -713,19 +713,19 @@ export class AuthenticationService extends Disposable implements IAuthentication
 	}
 
 	async getSessions(id: string, scopes?: string[], activateImmediate: boolean = false): Promise<ReadonlyArray<AuthenticationSession>> {
-		try {
-			const authProvider = this._authenticationProviders.get(id) || await this.tryActivateProvider(id, activateImmediate);
+		const authProvider = this._authenticationProviders.get(id) || await this.tryActivateProvider(id, activateImmediate);
+		if (authProvider) {
 			return await authProvider.getSessions(scopes);
-		} catch (_) {
+		} else {
 			throw new Error(`No authentication provider '${id}' is currently registered.`);
 		}
```

Comment:
I would rather you leave the try/catch in here and then switch between Error messages to show than get rid of this try/catch and expose the underlying error. It's not clear if the underlying error will always have readable messages that the user should see.